Documentation/running-lkvm-stage1.md

```diff
@@ -83,9 +83,9 @@ If you want to run software that requires hypervisor isolation along with truste
 For example, if you have a container stage1 named `stage1-coreos.aci` and a lkvm stage1 named `stage1-kvm.aci` in `/usr/local/rkt/`:
 
 ```
-# rkt run --stage1-image=/usr/local/rkt/stage1-coreos.aci coreos.com/etcd:v2.0.9
+# rkt run --stage1-path=/usr/local/rkt/stage1-coreos.aci coreos.com/etcd:v2.0.9
 ...
-# rkt run --stage1-image=/usr/local/rkt/stage1-kvm.aci coreos.com/etcd:v2.0.9
+# rkt run --stage1-path=/usr/local/rkt/stage1-kvm.aci coreos.com/etcd:v2.0.9
```

Comment:
These were paths before, so I used `--stage1-path`. Also, using `--stage1-name` implies discovering the image, which is probably not what we want in this case. (BTW. it is `--stage1-name=coreos.com/rkt/stage1-kvm`).